java/vector/src/test/java/org/apache/arrow/vector/types/pojo/TestExtensionType.java

```diff
@@ -204,6 +205,14 @@ public UUID getObject(int index) {
       return new UUID(bb.getLong(), bb.getLong());
     }
 
+    @Override public int hashCode(int index) {
+      return 0;
+    }
+
+    @Override public boolean equals(int index, ValueVector to, int toIndex) {
+      return false;
```

Comment:
I think by default extensiontype should delegate to its underlying valuevector for hash/equality

Reply:
fixed.